apps/api/src/app/admin/admin.service.ts

```diff
@@ -427,8 +427,17 @@ export class AdminService {
     };
   }
 
-  public async getUsers(): Promise<AdminUsers> {
-    return { users: await this.getUsersWithAnalytics() };
+  public async getUsers({ skip, take, sortColumn, sortDirection }: {
+    skip?: number;
+    take?: number;
+    sortColumn?: string;
+    sortDirection?: Prisma.SortOrder;
+  }): Promise<AdminUsers> {
+    return this.userRepository.find({
```

Comment:
Still use `return { users: await this.getUsersWithAnalytics() };`, but pass `skip`, `take` and integrate them in the query.